chain/network/src/peer/peer_actor.rs

```diff
@@ -1192,6 +1193,7 @@ impl PeerActor {
                                 && cur_edge.nonce() >= edge_info.nonce =>
                         {
                             // Found a newer local edge, so just send it to the peer.
+                            // TODO: need to implement some equivalent behavior for V2 routing
```

Comment:
I thought this was necessary originally, but the design has evolved so that we are for now treating the V1 graph as the source of truth for the state of the local edges. Eventually when we deprecate it in favor of v2, we'll need to touch this code.

Reply:
Nice, catch, it is an outdated comment. We are treating the V1 graph as the ultimate source of truth for local edge state, so there is no need to touch the V2 graph here. Eventually when we deprecate the V1 graph we'll touch this code.